saleor/graphql/schema.graphql

```diff
@@ -13240,6 +13240,13 @@ type TransactionEvent implements Node @doc(category: "Payments") {
   Added in Saleor 3.14.
   """
   idempotencyKey: String
+
+  """
+  Determines if the event was created automatically.
```

Comment:
non blocking: Can we add a little more? From integrator perspective I'd probably not understand what does it mean. We can do something like this:

```
Determines if the event was created automatically.

Some events will be created by Saleor, if not created by the app. For example, if first event reported by app is "CHARGE_SUCCESS", Saleor will add preceding event "CHARGE_REQUEST". 

This field can be used to differently show events that origin from the app from the ones created in the background
```